DEVELOPMENT.md

```diff
@@ -80,3 +80,12 @@ Starting application at 127.0.0.1:3000
 
 And you'll be up and running. As you edit and save code, the `buffalo dev` command will notice and automatically
 re-compile and restart the server.
+
+# Run unit tests
+
+In order to run unit tests dependencies need to be run first:
```

Comment:
Maybe this could be more clear as:
`In order to run unit tests, services they depend on must be running first:`
```console
make alldeps
```
`Then you can run the unit tests:`
```console
make test-unit
```

Reply:
67460ed